### PR TITLE
Fixed spelling errors and unclear wordings in /api and /api-2.0

### PR DESCRIPTION
**Pull request by alexianchen**

### CODE REVIEW

**Comment by robbyoconnor** [comment]
@mogoodrich

File: api-2.0/src/main/java/org/openmrs/module/htmlformentry/PatientServiceCompatibility2_0.java [hunk 1]
```diff
@@ -28,6 +28,6 @@
 
 	@Override
 	public void exitFromCare(Patient patient, Date dateExited, Concept reasonForExit) throws APIException {
-		//do nothing because this was moved to the exit from care module
+		//do nothing because this was moved to the exit from care module (remove?)
```
Comment (robbyoconnor) [inline api-2.0/src/main/java/org/openmrs/module/htmlformentry/PatientServiceCompatibility2_0.java]: Why is `{remove?}` written here? Were you suggesting to remove this?

**Comment by robbyoconnor** [comment]
Consider remove your formatting changes from this PR and rebasing and then making a separate PR for that?

File: api-2.0/src/main/java/org/openmrs/module/htmlformentry/PatientServiceCompatibility2_0.java (hunk 1 above)
Comment (alexianchen) [inline api-2.0/src/main/java/org/openmrs/module/htmlformentry/PatientServiceCompatibility2_0.java]: Yeah! it was "moved to the exit from care module", so shouldn't this be removed now as its redundant?

**Comment by robbyoconnor** [comment]
You dont need to close this 

Also, use feature branches!!!!!

**Comment by robbyoconnor** [comment]
Well actually you do -- you should use feature branches.

**Pull request closed without merging**